components/gc-subway/_screen-sm-max.scss

```diff
@@ -2,15 +2,24 @@
  * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
  * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
  *
- * Small view and under (screen only) - Variable $screen-sm-max <= 991 px
- *
- *	Provisional GC Subway map menu
+ * Small view and under (screen only) - Variable $screen-sm-max <= 991px
 */
 
-.provisional {
-	&.gc-subway {
-		dt, li {
-			max-width: initial;
+.gc-subway.provisional {
+	&:not(.gc-subway-index) {
+		h1 {
+			background-color: #fff;
+			border-bottom: none;
+			color: #555;
+			float: left;
+			font-size: 1.3em;
+			margin-left: -.5em;
+			margin-top: -.75em;
+			padding: 0px 20px 10px 0px;
+		}
+
+		ul {
+			padding-top: .5em;
```

Comment:
Is this needed?  It adds a "hat" to the first item in mobile, removing it still has a "hat" but smaller

Reply:
@Christopher-O  I actually think it looks better that way.

Reply:
@Garneauma I was mentioning to someone that out of all the comments this one was the only one I wouldn't mind as much if it stayed.  However, it does creates more inconsistency with the desktop version that does not have that "hat".

Reply:
@Christopher-O  I see your point. However, the mobile version also has a border that goes around the whole nav, so I don't see this "hat" as an inconsistency. Another option would be to remove that border altogether on mobile, that way it would really be consistent with the desktop version.

Reply:
@Garneauma I'm okay with leaving that in if it's agreed upon.